lib/shared/types.ts

```diff
@@ -664,20 +664,24 @@ export let version: string
 
 // Call this function to terminate esbuild's child process. The child process
 // is not terminated and re-created after each API call because it's more
-// efficient to keep it around when there are multiple API calls.
+// efficient to keep it around when there are multiple API calls. This child
+// process normally exits automatically when the parent process exits, so you
+// usually don't need to call this function.
 //
-// In node this happens automatically before the parent node process exits. So
-// you only need to call this if you know you will not make any more esbuild
-// API calls and you want to clean up resources.
-//
-// Unlike node, Deno lacks the necessary APIs to clean up child processes
-// automatically. You must manually call stop() in Deno when you're done
-// using esbuild or Deno will continue running forever.
+// One reason you might want to call this is if you know you will not make any
+// more esbuild API calls and you want to clean up resources (since the esbuild
+// child process takes up some memory even when idle).
 //
 // Another reason you might want to call this is if you are using esbuild from
 // within a Deno test. Deno fails tests that create a child process without
 // killing it before the test ends, so you have to call this function (and
-// await the returned promise) in every Deno test that uses esbuild.
+// await the returned promise) in every Deno test that starts esbuild.
+//
+// You may also start esbuild once at the top level of your test suite (by
+// calling `initialize()`) instead of starting and stopping the esbuild process
+// for every test. This will not interfere with the resource sanitizer, and will
+// improve the efficiency of your tests because the esbuild process can be
+// reused between tests. 
```

Comment:
I have added this paragraph to explain that you may also start esbuild outside of a `Deno.test` call to avoid the resource sanitizer flagging it.